src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,89 @@
-export const Pagination = () => {};
+import React from 'react';
+import classNames from 'classnames';
+
+type Props = {
+  items: string[];
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  items,
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const handleSelectPage = (page: number) => {
+    if (currentPage !== page) {
+      onPageChange(page);
+    }
+  };
+
+  const handleClickRight = () => {
+    if (currentPage !== 0) {
```

Comment:
The condition in the 'handleClickRight' function is incorrect. You should check if the current page isn't the last one, not if it's not zero.